docs/cloud/managing-airbyte-cloud/manage-airbyte-cloud-notifications.md

```diff
@@ -2,26 +2,35 @@
 
 This page provides guidance on how to manage notifications for Airbyte Cloud, allowing you to stay up-to-date on the activities in your workspace. 
 
-## Set up Webhook notifications
+
+## Configure Notification Settings
 
 To set up Webhook notifications:
 
 1. On the [Airbyte Cloud](http://cloud.airbyte.com) dashboard, click **Settings**.
 
 2. Click **Notifications**.
 
-3. [Create an Incoming Webhook for Slack](https://api.slack.com/messaging/webhooks).
+3. Have a webhook URL ready if you plan to use webhook notifications. Using a Slack webhook is recommended. [Create an Incoming Webhook for Slack](https://api.slack.com/messaging/webhooks).
+
+4. Toggle the type of events you are interested to receive notifications for. 
+	1. If using slack webhooks, the webhook URL is required. For your convenience, we provide a 'test' function to send a test message to your slack channel so you can make sure it's working as expected.
```

Comment:
Wording suggestion: To enable webhook notifications, the webhook URL is required. For your convenience, we provide a 'test' function to send a test message to your webhook URL so you can make sure it's working as expected.

Could you also add another sub-bullet that says: "Enabled email notifications will send an email to all users of the workspace."

Reply:
On the note about email notifications - that's my understanding of how this will work but please correct me if I'm wrong!

Reply:
Right - no, email notifications at this moment will only be sent to the owner of the workspace.

Reply:
Oh interesting. Okay thanks.